scripts/trello/api.js

```diff
@@ -0,0 +1,19 @@
+async function get(relativeUrl, params, { trelloApiKey, trelloOAuth1 } = {}) {
+    const res = await axios.get(buildUrlWithParams(trelloApiKey, trelloOAuth1, relativeUrl, params));
+    return res.data;
+}
+
+async function post(relativeUrl, params, { trelloApiKey, trelloOAuth1 } = {}) {
+    const res = await axios.post(buildUrlWithParams(trelloApiKey, trelloOAuth1, relativeUrl, params));
+    return res.data;
+}
+
+async function put(relativeUrl, params, { trelloApiKey, trelloOAuth1 } = {}) {
+    const res = await axios.put(buildUrlWithParams(trelloApiKey, trelloOAuth1, relativeUrl, params));
+    return res.data;
+}
+
+function buildUrlWithParams(trelloApiKey, trelloOAuth1, relativeUrl, params) {
```

Comment:
@AdriAt360 Old school tip if you want to create package without a bundler:
you can wrap it in a IIFE
```javascript
const { exposedElement } = (() => {
    ... private definitions
    return { exposedElement };
})()` // <- fork bomb style
```

Will make the whole file cleaner, and will hide this method from global scope.

Reply:
@pp0rtal I'm not sure I will do that yet, for readability's sake in this file (ok, I'll keep these functions global, but there are no consequences), but maybe in the next PR
when you say "create package", do you mean that I would be able to `import` directly stuff in the other files? Seems tough as the `import` must be done inside a module, but if so, I might do that in the next ticket

Reply:
@AdriAt360 My suggestion was about exposing function which needed to be exposed. As here, any function is global. Wrapping them into an IIFE can make your private functions have the file scope.

I don't know about import/export on front-end side, I didn't code on front-end for years (or only on stacked project) :smile: